desktop/apps/artwork/client/fold.jade

```diff
@@ -2,4 +2,5 @@ include ../components/partner/index
 include ../components/show_artworks/index
 include ../components/artist_artworks/index
 include ../components/auction_artworks/index
+#react-mount-current-auctions
```

Comment:
@sweir27 - interested to brainstorm some possible ways of approaching this considering SSR, etc